keps/sig-testing/2420-reducing-kubernetes-build-maintenance/kep.yml

```diff
@@ -0,0 +1,46 @@
+title: KEP Template
+kep-number: 2420
+authors:
+  - "BenTheElder"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: yyyy-mm-dd
+reviewers:
+  - dims
+  - liggitt
+approvers:
+  - spiffxp
+  - justaugustus
+prr-approvers:
+  - TBD
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha|beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.19"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
```

Comment:
we will need to fix these milestones @BenTheElder

Reply:
I put in something provisionally, but I think beta/GA are slightly TBD: https://github.com/kubernetes/enhancements/pull/2421#discussion_r570476432

Reply:
@justaugustus can we delete the alpha/beta milestones or can they be listed as "n/a"?